src/templates/_macros/feed_websites.html

```diff
@@ -0,0 +1,57 @@
+{% from '_macros/app_tile.html' import deferred_icon %}
+{% from "_macros/market_button.html" import market_button %}
+{% from "_macros/previews.html" import preview_tray %}
+{% from "_macros/stars.html" import stars %}
+
+{% macro website_tabs(websites) %}
+  {% if websites %}
+    <section class="feed-item-item feed-item-website feed-item-website-tabs">
+      <ol>
+        {% for website in websites %}
+          {% set website = apps.transform(website) %}
+          <li class="mkt-tile">
+            <a href="#" class="feed-item-website-tabs-tab">
```

Comment:
Don't need the `href` here which will probably save you a `evt.preventDefault()` later.